doc/design/record_fetcher_proposal.md

```diff
@@ -77,22 +77,18 @@ can use to start that conversation
 * should probably losslessly compress most things (we get 40% savings with gzip on the sample manifest below)
     
 ## Design Ideas
+
 ### Document Storage
-First, let's talk document storage. Have record fetchers crawl however they want and upload documents to a document
-storage endpoint. This should ideally be some kind of content-addressed blob storage service. We don't need a lot of
-structure for this, as any relationship between documents will be encoded by the manifests -- the service just needs to
-be able to store the raw documents for later lookup. Backing this storage API with a fairly flat filesystem is fine
-here, we're rarely going to list files, only write and read. If we do need to list files, it'll be for some cleanup
-operation and should not be on any request path.
+First, let's talk document storage. This will be a content digest-addressed blob storage service. We don't need a
+lot of structure for this, as any relationship between documents will be encoded by the manifests -- the service just
+needs to be able to store the raw documents for later lookup (but lookups don't need to be via this service). Backing
+this storage service with a fairly flat filesystem is fine here, we're rarely going to list files, only write and read.
 
 A big benefit of content-addressed storage here is that we get deduping almost for free -- we may need to zero out
```

Comment:
I'd posit that we'd want to keep a record of material changes, which this makes difficult. Raw docs stored for this would be expensive from a storage standpoint, but diffs could be an interesting alternative.